include/veriblock/logger/logger.hpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2019-2020 Xenios SEZC
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ALT_INTEGRATION_LOGGER_LOGGER_HPP
+#define ALT_INTEGRATION_LOGGER_LOGGER_HPP
+
+#define FMT_HEADER_ONLY 1
+
+#include <veriblock/third_party/fmt/printf.h>
+
+#include <memory>
+#include <string>
+
+namespace altintegration {
+
+class Logger {
+ public:
+  virtual ~Logger() = default;
+  virtual void debug(std::string msg) = 0;
+  virtual void info(std::string msg) = 0;
+  virtual void warn(std::string msg) = 0;
+  virtual void error(std::string msg) = 0;
```

Comment:
I think you could add default implementation here (e.g. remove =0 and put empty impl there)

Reply:
Also, maybe pass string by const ref?